src/styles/themes/_default-theme.scss

```diff
@@ -10,12 +10,23 @@
     --primary-color-90: #002a47;
     --primary-color-100: #00060a;
     --primary-color: #146da6;
+    --accent-color-10: #ebfdff;
```

Comment:
From Yair:

The color names that ends with
80 are primary colors (all pass AAA)
70 are secondary colors (texts pass accessibility on white bg)
60 are tertiary colors (images pass accessibility on white bg)
30 are background 3 colors
20 are background 2 colors
10 are background 1 colors

This means that the blue hexes are the primary palette. Please remove the --accent-color-... and replace the --primary-color-... values with those hexes. Then we should be good to go.